cumulusci/cli/org.py

```diff
@@ -319,13 +324,26 @@ def org_list(runtime, plain):
                 [org_days, not org_config.active, org_config.config_name, domain]
             )
             scratch_data.append(row)
+            json_data[org] = {
+                "isDefault": is_org_default,
+                "days": org_days,
+                "expired": not org_config.active,
+                "config": org_config.config_name,
+                "domain": domain,
+                "isScratch": True,
+            }
         else:
             username = org_config.config.get(
                 "username", org_config.userinfo__preferred_username
             )
             row.append(username)
             row.append(org_config.expires or "Unknown")
             persistent_data.append(row)
+            json_data[org] = {"isDefault": is_org_default, "isScratch": False}
```

Comment:
Can we use is_default and is_scratch? Javascript/Typescript tend to use camel case for variable names and keys in a mapping, but Python tends to use snake case (underscores) and CumulusCI generally follows that convention.